projects/pastanaga-angular/src/lib/controls/textfield/select/select.component.ts

```diff
@@ -52,6 +52,13 @@ export class SelectComponent extends PaFormControlDirective implements OnChanges
     @Input() help?: string;
     @Input() errorMessages?: IErrorMessages;
     @Input() showAllErrors = true;
+    _dim = false;
```

Comment:
`_dim` should be private (and usually we group private members after inputs and outputs)